packages/docs/page-config/ui-elements/data-table/examples/Selection.vue

```diff
@@ -107,15 +102,15 @@ export default defineComponent({
     ];
 
     return {
-      items: users,
+      items,
       columns,
       selectable: true,
-      selectedItems: [users[1]],
+      selectedItems: [items[1]],
       selectedItemsEmitted: [],
       selectMode: "multiple",
-      selectedColor: "#888888",
+      selectedColor: "#EF467F",
```

Comment:
Let's have a primary by default, so people will not be scared of pink here.
```suggestion
      selectedColor: "primary",
```